dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/FunctionChoiceBehaviorConfigurationContext.cs

```diff
@@ -10,6 +11,20 @@ namespace Microsoft.SemanticKernel;
 [Experimental("SKEXP0001")]
 public sealed class FunctionChoiceBehaviorConfigurationContext
 {
+    /// <summary>
+    /// Creates a new instance of <see cref="FunctionChoiceBehaviorConfigurationContext"/>.
+    /// </summary>
+    /// <param name="chatHistory">The chat history.</param>
+    public FunctionChoiceBehaviorConfigurationContext(ChatHistory chatHistory)
+    {
+        this.ChatHistory = chatHistory;
+    }
+
+    /// <summary>
+    /// The chat history.
```

Comment:
Suggest adding more where this chat history comes from in the context.

```suggestion
    /// The chat history.
```

Reply:
yep, make sense, done.

Reply:
fixed, thnaks.